pkgs/tools/misc/wasm-tools/add-Cargo.lock.patch

```diff
@@ -0,0 +1,2168 @@
+diff --git a/Cargo.lock b/Cargo.lock
```

Comment:
please vendor the `Cargo.lock` directly and use

```nix
{
  # ...
  cargoLock = {
    lockFile = ./Cargo.lock;
  };
  # ...
}
```

another option is to use the crate version, which has `Cargo.lock` but doesn't have submodules

```nix
fetchCrate {
  inherit pname version;
  hash = "sha256-2C1HT8B5AEWiv5sDJmXPpAPJ+3qGLYiHJLNmuooNElI=";
}
```